mnefun/_report.py

```diff
@@ -251,6 +252,37 @@ def _report_raw_psd(report, raw, raw_pca=None, p=None):
     print('%5.1f sec' % ((time.time() - t0),))
 
 
+def _peak_times(evoked, max_peaks=5):
+    '''Return times of prominent peaks from global field power.'''
+    gfp_list = []
+    if 'meg' in evoked:
+        evk = evoked.copy().pick_types(meg=True)
+        gfp = evk.data.std(axis=0)
+        gfp_list.append(gfp/gfp.max())
+    if 'eeg' in evoked:
+        evk = evoked.copy().pick_types(meg=False, eeg=True)
+        gfp = evk.data.std(axis=0)
+        gfp_list.append(gfp/gfp.max())
+    if not gfp_list:    # for non-meg/eeg data types
+        gfp = evoked.data.std(axis=0)
+        gfp = gfp/gfp.max()
+    else:
+        gfp = np.array(gfp_list).mean(axis=0)
+                        # no less than 1, no more than max_peaks
+    npeaks = max(min(len(gfp)//3, max_peaks), 1)
+    peaks = find_peaks(gfp)[0]
+    prms = peak_prominences(gfp, peaks)[0]
+    times = peaks[prms.argsort()[::-1]][:npeaks]
+    times.sort()
+    
+    if not len(times):  # guarantee at least 1
+        times = gfp.argmax()
+    times = evoked.times[times]  # convert units to seconds
+    
+    print('Local peak times calculated at', times, 'sec.')
```

Comment:
That wasn't intended for printing to the .html, only to the console (like the other print statements (e.g. `print('    %s skipped' % section)`), but I'll try it.

Reply:
Yeah it won't end up in the HTML, but if you run the report generation you should see it nicely formatted with the operation and timing, and with this `print` statement in there it will mess it up